chatrooms.lua

```diff
@@ -93,14 +101,37 @@ function ToArrival( tUser, sMessage )
 		Core.SendPmToUser( tUser, sTo, "Sorry! You don't have access to the chatroom.|" )
 		return true
 	else
-		local sChat = sMessage:match "%b$$(.*)|"
-		SendToRoom( tUser, sTo, sChat )
+		local sChat = sMessage:match "%b$$(.*)|" 
+		SaveToFile(sChat, sTo)
```

Comment:
Blank spaces around parameters. Good decision about moving this out of `SendToRoom` definition.